ph5/utilities/resp_load.py

```diff
@@ -362,91 +439,85 @@ def load_response(self, path, nickname, data, input_csv):
                             LOGGER.info(
                                 "Loaded {0}"
                                 .format(name.replace(" ", "")))
-                        except BaseException:
-                            LOGGER.warning(
-                                "Could not load {0}"
-                                .format(name.replace(" ", "")))
+                        except BaseException as e:
+                            if "already has a child" in str(e):
```

Comment:
Does this mean you can only load a resp once?
Shouldn't subsequent loads, replace, not error?

Reply:
It can have subsequent loads, but cannot be replace. However, I am going to commit another version of resp_load that give user an option to reload response data.

Reply:
I think it would also be helpful to have an option to load a response file without automatically updating the response_t table. Also is there a way to remove a response once it has been loaded?